### ISSUE TITLE
grab not released on mouse up outside dragdroplist

### ISSUE THREAD

**Issue by helbling** [jwlarocque/svelte-dragdroplist#6]
steps to reproduce:
- grab an element using the mouse
- move pointer outside dragdroplist
- release mouse press

expected:
- element would be released where it last was moved
- element should not follow the pointer again when moving the pointer back into dragdroplist

what happens instead:
- element is still in grabbed state
- follows the pointer again when moving the pointer back into dragdroplist

edit: tested with desktop Chrome and Firefox using a mouse, seems to work fine with touch events on mobile Chrome and Firefox

**Comment by ianengelbrecht** [comment]
Yes, I got this too on the simple example repl. Drag an item to the bottom of the list and drop and it 'sticks' to the mouse pointer.

**Comment by jamersoniiv** [comment]
I get this too, has anyone found a fix or workaround?

**Comment by Roger-Sa** [comment]
Adding this in the component works:
<svelte:window on:mouseup={release} on:touchend={release} />

https://svelte.dev/repl/03dc6af80d0e4507abbbe80aad079fa5?version=3.43.1

**Comment by jamersoniiv** [comment]
> Adding this in the component works: <svelte:window on:mouseup={release} on:touchend={release} />
> 
> https://svelte.dev/repl/03dc6af80d0e4507abbbe80aad079fa5?version=3.43.1

Hice Roger-Sa, I had added an on:mouseleave event to the List div, which worked to cancel the drag when you left the div, but your solution is much more as I would expect it to behave.

So thanks for sharing!
